src/test/java/duke/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.tasks;
+
+import duke.exceptions.DukeException;
+import duke.util.DateFormatter;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests Deadline creation.
+ */
+public class DeadlineTest {
+    @Test
+    public void testDeadlineCreation() {
```

Comment:
Perhaps you could use the three part format ```featureUnderTest_testScenario_expectedBehavior()``` for test method names? The three part format provides more clarity on what is being tested in the method.

This three part format naming convention can also be applied for your other test method names.

Reply:
Thanks! I'll to switch to those soon.